plio/tests.py

```diff
@@ -268,3 +652,19 @@ def test_each_image_has_unique_name(self):
         random.seed(10)
         test_image_2 = Image.objects.create()
         self.assertNotEqual(test_image_1.url.name, test_image_2.url.name)
+
+    def test_duplicate_image(self):
+        """Tests the duplicate functionality for images"""
+        # duplicate image
+        response = self.client.post(f"/api/v1/images/{self.image}/duplicate/")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        image = Image.objects.filter(id=self.image).first()
+        new_image = Image.objects.filter(id=response.data["id"]).first()
+
+        self.assertNotEqual(self.image, response.data["id"])
+        self.assertEqual(image.alt_text, response.data["alt_text"])
+        self.assertEqual(image.url.file.size, new_image.url.file.size)
+
+    # before merge, update Vaibhav's changes for org and new user - use
```

Comment:
this has been kept as a reminder based on whether Vaibhav's or this PR gets merged first. Will remove it before merging.